audformat/core/scheme.py

```diff
@@ -44,6 +65,17 @@ class Scheme(HeaderBase):
         {dtype: int}
         >>> Scheme(float, minimum=0, maximum=1)
         {dtype: float, minimum: 0, maximum: 1}
+        >>> # Misc Table as Scheme
+        >>> import audformat
+        >>> db = audformat.Database('mydb')
+        >>> db.schemes['age'] = Scheme('int')
+        >>> db['speaker'] = audformat.MiscTable(
+        ...     pd.Index(['spk1', 'spk2'], name='speaker')
+        ... )
+        >>> db['speaker']['age'] = audformat.Column(scheme_id='age')
```

Comment:
I think would simplify the example and use a `MiscTable` without a column.

Reply:
Yes, I think this is something we should cover in the usage section, for this example I find it too complicated.

Reply:
OK, I removed the column and the `age` scheme and updated the description of this pull request.